portal/src/components/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "gatsby";
+import React from "react";
+
+import "./Header.scss";
+
+interface Props {
+    siteTitle?: string;
+}
+
+export const Header = ({ siteTitle = "" }: Props) => (
+    <header className="portal-header">
+        <h1 className="portal-header__title jkl-h2" style={{}}>
```

Comment:
empty style?

Reply:
Yeah, I forgot the small fixes in the middle of my rebase-anxiety, haha!